lib/config.js

```diff
@@ -122,6 +129,14 @@ function init() {
         "level.co2",
         "id_text_co2_limit"
     );
+	
+	number(vlxObjects,
+        "A_CYC_CO2_THRESHOLD",
+        "level.co2",
+        "id_text_co2_limit",
+        "",
```

Comment:
can you add the unit for co2? ("ppm")
Would be nice if you could add the unit to "A_CYC_CO2_VALUE", too.

Reply:
yes, i fix it now